a/ /lib/systemd/system/coolwsd.service
b/ /lib/systemd/system/coolwsd.service
@@ -4,7 +4,7 @@
 
 [Service]
 EnvironmentFile=-/etc/sysconfig/coolwsd
-ExecStart=/usr/bin/coolwsd --version --o:sys_template_path=/opt/cool/systemplate --o:child_root_path=/opt/cool/child-roots --o:file_server_root_path=/usr/share/coolwsd --o:cache_files.path=/opt/cool/cache
+ExecStart=/usr/bin/coolwsd --version --o:sys_template_path=/opt/cool/systemplate --o:child_root_path=/opt/cool/child-roots --o:file_server_root_path=/usr/share/coolwsd --o:cache_files.path=/opt/cool/cache  --port __PORT__
 KillSignal=SIGINT
 TimeoutStopSec=120
 User=cool
